src/test/java/AllSongsTest.java

```diff
@@ -0,0 +1,75 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AllSongsPage;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class  AllSongsTest extends BaseTest{
+
+    @Test
+    public void  playWithContextClick() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.